crates/cli/Cargo.toml

```diff
@@ -32,6 +32,7 @@ release_channel.workspace = true
 serde.workspace = true
 util.workspace = true
 tempfile.workspace = true
+tokio = { workspace = true, features = ["full"] }
```

Comment:
I really do not understand the whole `async` story in this PR:

* `load_shell_from_passwd` does not deal with asynchronicity at all, why is it even `async`?
*  Why does `load_login_shell_environment` use a `smol::Command` instead of the stdlib one? Esp. given that it's called in the `main` methods, in the very beginning.

So far, seems that everything can be synchronous all around.

At worst, we definitely do not need the `full` tokio in the CLI, but I think we should have none at all.

Reply:
That makes sense. I was also curious why it's async but didn't delve deeper into the function implementation. I'll make this change.